netstandard/ref/System.Threading.Tasks.cs

```diff
@@ -434,4 +434,47 @@ public partial class UnobservedTaskExceptionEventArgs : System.EventArgs
         public bool Observed { get { throw null; } }
         public void SetObserved() { }
     }
+    [System.Runtime.CompilerServices.AsyncMethodBuilderAttribute(typeof(System.Runtime.CompilerServices.AsyncValueTaskMethodBuilder))]
```

Comment:
This looks like redundant

Reply:
@marek-safar As I understand it, `AsyncMethodBuilderAttribute` is used by the C# compiler to figure out how to generate an `async` method that returns the given type (here, `ValueTask`). So I think it should be included in reference assemblies.

Reply:
Yes, it's required. The compiler uses it to determine what builder type to use for async methods that return the annotated type.